src/index.html

```diff
@@ -1,17 +1,209 @@
+            </label>
+          </li>
+        </ul>
+      </fieldset>
+
+      <fieldset class="form__field">
+        <legend>An interesting fact about you!</legend>
+        <ul class="field__list">
+          <li class="list__item">
+            <label for="cars_yes">
+              Do you love cars?
+              <input 
+                type="radio" 
+                name="love_cars" 
+                id="cars_yes"
+              >
+              Yes
+            </label>
+            <label for="cars_no">
+              <input type="radio" 
+                name="love_cars" 
+                id="cars_no"
+              >
+              No
+            </label>
+          </li>
+          <li class="list__item">
+            <label for="color">
+              What is your favorite color?
+              <input type="color" 
+                name="color" 
+                id="color"
+              >
+            </label>
+          </li>
+          <li class="list__item">
+            <label for="time">
+              What time do you go to bed?
+              <input 
+                type="time" 
+                name="time" 
+                id="time"
+              >
+            </label>
+          </li>
+          <li class="list__item">
+            <label for="car">
+              What are your favorite brand of cars?
+              <select 
+                type="text" 
+                name="car" 
+                id="car"
+                multiple
+              >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+              </select>
+            </label>
+            
+          </li>
+          <li class="list__item">
+            <label for="rate">
+              How do you rate our work?
+              <input 
+                type="range" 
+                min="0" 
+                max="10" 
+                name="rate" 
+                id="rate"
+              >
+            </label>
+          </li>
+        </ul>
+      </fieldset>
+
+      <fieldset class="form__field">
+        <legend>Addictional info:</legend>
+        <ul class="field__list">
+          <li class="list__item">
+            <label for="comments">
+              Comments:
+              <textarea 
+                name="comments" 
+                id="comments" 
+                cols="20" 
+                rows="5"
+              >
+            </textarea>
+            </label>
+          </li>
+          <li class="list__item">
+            <label for="recommend">
+              Would you recommend us?
+              <select name="recommend" id="recommend">
+                <option value="no">no</option>
+                <option value="yes">yes</option>
+              </select>
+            </label>
+          </li>
+        </ul>
+      </fieldset>
+    </form>
+    <button type="submit">Submit</button>
+  </main>
+</body>
+
+</html>
```

Comment:
```suggestion
</html>

```
Always add an empty line at the end of the document